docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -401,7 +401,7 @@ Remove both Sam and Sig..
 		'Surname' => 'Minnée',
 	));
 
-And removing Sig and Sam with that are either age 17 or 43.
+And removing Sig and Sam with that are either age 17 or 74.
```

Comment:
Something wrong here, it's a reverse of https://github.com/silverstripe/silverstripe-framework/pull/4000/files